src/sonic-yang-models/tests/yang_model_tests/tests_config/bgp.json

```diff
@@ -1322,8 +1322,87 @@
                 ]
             }
         }
-    }
+    },
 
+    "BGP_MONITORS_ALL_VALID": {
+        "sonic-bgp-monitor:sonic-bgp-monitor": {
+            "sonic-bgp-monitor:BGP_MONITORS": {
+                "BGP_MONITORS_LIST": [
+                    {
+                        "addr": "10.0.0.1",
+                        "asn": "65200",
+                        "holdtime": "180",
+                        "keepalive": "60",
+                        "local_addr": "10.0.0.2",
+                        "name":"BGPMonitor",
+                        "nhopself":"0",
+                        "rrclient":"0",
+                        "admin_status":"up"
+                    }
+                ]
+            }
+        }
+    },
+
+    "BGP_MONITORS_NEG_INVALID_ADMIN_STATUS": {
+        "sonic-bgp-monitor:sonic-bgp-monitor": {
+            "sonic-bgp-monitor:BGP_MONITORS": {
+                "BGP_MONITORS_LIST": [
+                    {
+                        "addr": "10.0.0.1",
+                        "asn": "65200",
+                        "holdtime": "180",
+                        "keepalive": "60",
+                        "local_addr": "10.0.0.2",
+                        "name":"BGPMonitor",
+                        "nhopself":"0",
+                        "rrclient":"0",
+                        "admin_status":"left"
+                    }
+                ]
+            }
+        }
+    },
+
+    "BGP_MONITORS_NEG_INVALID_NAME": {
+        "sonic-bgp-monitor:sonic-bgp-monitor": {
+            "sonic-bgp-monitor:BGP_MONITORS": {
+                "BGP_MONITORS_LIST": [
+                    {
+                        "addr": "10.0.0.1",
+                        "asn": "65200",
+                        "holdtime": "180",
+                        "keepalive": "60",
+                        "local_addr": "10.0.0.2",
+                        "name":"BGPMonitor123",
```

Comment:
Does this test case validate 'name' field? having the digit in the name is invalid?

Reply:
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/minigraph.py#L823
It seems that the name must be 'BGPMonitor'